Solutions/Darktrace/Analytic Rules/CreateAlertFromSystemStatus.yaml

```diff
@@ -14,20 +14,43 @@ triggerThreshold: 0
 tactics: # none
 relevantTechniques: # none
 query: |
-  darktrace_model_alerts_CL
+  darktrace_model_alerts_CL //anything starting with 'Dt' is not an ASIM mapping 
   | where dtProduct_s =="System Alert"
-  | extend EventVendor = "Darktrace"
-  | extend EventProduct = "Enterprise Immune System"
+  | extend EventVendor="Darktrace", EventProduct="Darktrace DETECT"
+  | project-rename ThreatCategory=dtProduct_s, EventStartTime=time_s, NetworkRuleName=friendlyName_s, SrcIpAddr=deviceIP_s, SrcHostname=hostname_s, ThreatRiskLevel=priority_code_d, ThreatRiskCategory=priority_s, DtSeverity=Severity, DtName=name_s, DtStatus=status_s, DtMessage=Message, DtURL=url_s, DtUUID=uuid_g
 eventGroupingSettings:
   aggregationKind: AlertPerResult
-entityMappings: # none
+entityMappings:
+  - entityType: Host
+    fieldMappings:
+      - identifier: HostName
+        columnName: SrcHostname
+  - entityType: IP
+    fieldMappings:
+      - identifier: Address
+        columnName: SrcIpAddr
 customDetails:
-  Title: friendlyName_s
-  EventMessage: Message
+  EventStartTime: EventStartTime
+  NetworkRuleName: NetworkRuleName
+  ThreatRiskLevel: ThreatRiskLevel
+  ThreatRiskCategory: ThreatRiskCategory
+  DtName: DtName
+  DtStatus: DtStatus
+  DtMessage: DtMessage
+  DtSeverity: DtSeverity
 alertDetailsOverride:
-  alertDisplayNameFormat: 'Darktrace: {{friendlyName_s}}'
-  alertDescriptionFormat: "{{Message}}"
+  alertDisplayNameFormat: 'Darktrace: {{ThreatRiskLevel}} - {{NetworkRuleName}}'
+  alertDescriptionFormat: '{{DtMessage}}'
   alertTacticsColumnName: # none
   alertSeverityColumnName: # none 
-version: 1.0.0
-kind: NRT
```

Comment:
Any specific reason of converting this query to scheduled?

Reply:
We were having issues with System Status rule as NRT and we consider the System Status Alerts to be secondary in importance to Model Breaches and AI Analyst incidents as they surface meta-information about the Darktrace deployment. These are typically addressed outside of the SOC alert processing workflow and rather are tasks for network engineering teams or staff otherwise involved in the maintenance of Darktrace deployments. Seeing that they are less time-critical than the actual security alerting we are happy for this to run on a schedule rather than NRT.

The current package contains these rules as Scheduled so this element remains unchanged from previous versions therefore we believe it can be signed off.

Reply:
Thanks for the explanation.